### DISCUSSION TITLE
Pending Releases

### DISCUSSION THREAD

**Discussion by lrljoe** [rappasoft/laravel-livewire-tables#1186] [General]
To try and keep everyone advised of what is coming in future releases, I'll keep this updated with any PRs / overviews of upcoming fixes/features.

Expect this post to be modified!  The content will be available in the ChangeLog for the release.  Documentation fixes will not be included in this list.

For released features/fixes, please see the Change Log, this thread is only for pending releases: [CHANGELOG.md](https://github.com/rappasoft/laravel-livewire-tables/blob/master/CHANGELOG.md)

**Since 2.14**
### New Functionality
None yet

### Improvements
- Collapsing Columns migration to AlpineJS

### Fixes
- Fix for setHideBulkActionsWhen Empty

# Works In Progress
## On My Priority List (1-3 Releases)
- Option to AlwaysCollapse Certain Columns (Dependent on Migration to AlpineJS for Columns)
- Date Format for Date Filter
- Bulk Actions - Deselect On Page

## Medium Priority (Within next 3-5 Releases)
- ToggleColumn (Extension/Simplification of the Boolean with Clickable)

## Low Priority (5-10 Releases Time)(
- Filter ShowIf()
   This one is taking a bit more care as we need to ensure that no default values pass into it etc.

## Back Burner
- Additional Collapsing Breakpoints
  This one may well need to wait until v3

**Comment by lrljoe** [comment]
For released features/fixes, please see the Change Log, this thread is only for pending releases: [CHANGELOG.md](https://github.com/rappasoft/laravel-livewire-tables/blob/master/CHANGELOG.md)

**Comment by happymacarts** [comment]
Will Collapsing Columns migration to AlpineJS have the ability to collapse on other breakpoints as well?

**Reply by lrljoe** [reply]
Not in the first iteration, although it's certainly something that can be considered for a future version, particularly if there's a strong demand for it.

I'd suggest opening a feature request so demand can be assessed, and the request tracked
